stable/prometheus/templates/server-statefulset.yaml

```diff
@@ -9,6 +9,7 @@ metadata:
   labels:
     {{- include "prometheus.server.labels" . | nindent 4 }}
   name: {{ template "prometheus.server.fullname" . }}
+    {{ toYaml .Values.server.statefulSet.labels | indent 4 }}
```

Comment:
i think that this should move 1 line up

Reply:
Hmm if I remember correctly it worked for me let me double check then.

Reply:
Found it, I did not test if they were empty 🤦‍♂ my bad.